src/aggregation/agg_req_with_accessor.rs

```diff
@@ -321,6 +389,20 @@ fn get_ff_reader(
     Ok(ff_field_with_type)
 }
 
+fn get_dyn_column(
```

Comment:
```suggestion
fn get_dynamic_column(
```

dyn is a reserved word in rust